accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/analyzer/StatementAnalyzer.java

```diff
@@ -158,21 +160,28 @@ protected Scope visitTable(Table node, Optional<Scope> scope)
             analysis.addTable(tableName);
             if (tableName.getCatalogName().equals(accioMDL.getCatalog()) && tableName.getSchemaTableName().getSchemaName().equals(accioMDL.getSchema())) {
                 analysis.addModelNodeRef(NodeRef.of(node));
-                // only record model fields here, others are ignored
-                List<Field> modelFields = accioMDL.getModel(tableName.getSchemaTableName().getTableName())
-                        .map(Model::getColumns)
-                        .orElseGet(ImmutableList::of)
-                        .stream()
-                        .map(column ->
-                                Field.builder()
-                                        .modelName(tableName)
-                                        .columnName(column.getName())
-                                        .name(column.getName())
-                                        .build())
-                        .collect(toImmutableList());
+                List<Column> columns = new ArrayList<>();
```

Comment:
```suggestion
                List<Column> columns = ImmutableList.of();
```

Reply:
Already addressed comments in the latest commit.